tests/smoke/support/world/actor.ts

```diff
@@ -14,10 +14,6 @@ export class ActorContinent {
   }
 
   public async create({ id }: { id: string }): Promise<Actor> {
-    if (this.store.has(id)) {
-      throw new Error(`Actor '${id}' already exists.`)
```

Comment:
Why remove this code?

Reply:
please bring this back, if you do not want to fail use a try catch one level above, it is used for `following users have been created` and can be catched there, but i highly recommend to not do it! It takes care that everything is provisioned proper.

e.g.

```
try {
        await api.user.createUser({ user, admin })
      } catch (e) {
        console.warn(e)
      }
```

Reply:
I also found one problem. when you run several tests together or two test in one feature you get an error here

Reply:
Can confirm, stumbled upon the same thing today